lib/introspect.js

```diff
@@ -13,8 +13,16 @@ module.exports = function(obj, callback) {
           if (err) return callback(err);
           if (!result.interface) return callback(new Error('No such interface found'));
           var proxy = {};
+          var nodes = [];
           var i, m, s, ifaceName, method, property, signal, iface, a, arg, signature, currentIface;
           var ifaces = result['interface'];
+          var xmlnodes = result['node'];
+          if (xmlnodes == null) xmlnodes = [];
+
+          for (i = 1; i < xmlnodes.length; ++i) {       // Start at 1 because we want to skip the root node
+              nodes.push(xmlnodes[i]['@']['name']);
```

Comment:
I guess it'll break. Then again, a few lines later you do the same thing with `iface['@'].name`, so I didn't assume I'd have to put a check on that.